changelog/issue-7151.md

```diff
@@ -0,0 +1,6 @@
+audience: users
+level: major
```

Comment:
Why is this a major bump?

Reply:
good question.. maybe it isn't?
To me it felt like a breaking change, but it's not really breaking anything (unless it doesn't work on prod haha)

Reply:
😄 Definitely seemed more like a minor bump to me, unless I was missing some breaking change, but I don't think there is one.

Reply:
convinced :) Minor it is then!